README.md

```diff
@@ -82,10 +82,22 @@ so Docker does not need sudo permissions.
 - [Nomad](https://www.nomadproject.io/docs/install/index.html#precompiled-binaries) can be installed on Linux clients with `sudo ./install_nomad.sh`.
 - git-crypt
 - jq
+- iproute2
 
 Instructions for installing Docker and Nomad can be found by
-following the link for each service. git-crypt and jq can be installed via
-`sudo apt-get install git-crypt jq`.
+following the link for each service. git-crypt, jq, and iproute2 can be installed via
+`sudo apt-get install git-crypt jq iproute2`.
+
+When installing pip packages later in the install, you might get an error saying you need sudo permissions.
+In order to fix this you have to edit your `~/.config/pip/pip.conf` to add this:
+
+```
+[install]
+user = yes
+no-binary = :all:
+```
+
+This sets pip to install all packages in your user directory so sudo is not required for pip intalls.
```

Comment:
Was sudo necessary for installing some pip packages even within the virtualenvironment?

Reply:
The issues is when installing `virtualenv` itself. If the pip directory is in a place where only root has write access, `pip install virtualenv` fails, which can happen on some distros since the package manager wants complete control over installed software.